src/Components/HighChartsWrapper/HighChartsWrapper.tsx

```diff
@@ -7,13 +7,14 @@ import {
 } from './HighChartsWrapper.types';
 import { getStyles } from './HighChartsWrapper.styles';
 import { classNamesFunction, useTheme, styled } from '@fluentui/react';
-import Highcharts, {
+import {
     AlignValue,
     ColorString,
     DataGroupingApproximationValue,
     OptionsLayoutValue,
     SeriesOptionsType
 } from 'highcharts';
+import Highcharts from 'highcharts/highstock';
```

Comment:
it seems kind of odd that we wouldn't get the same thing here as we do from the root. What is the difference with the highstock folder? I saw your description in the PR but why do they have 2 versions of their library in the same package?

Reply:
it is not two versions, they are different libraries under the same package: https://api.highcharts.com/ 
![image](https://user-images.githubusercontent.com/45217314/198384590-fa55c899-2cb2-4b8e-9f97-c6f1f05dc334.png)